### PR TITLE
Plain language edits to Phase II application page

### PR DESCRIPTION
**Pull request by kategarklavs**
Edited this page to shift voice/tone & improve plain language.

Fixes issue(s) # .

[CircleCI](https://circleci.com/gh/18F/nsf-sbir/tree/BRANCH_NAME)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/nsf-sbir/BRANCH_NAME/)

[Preview README for this branch](https://github.com/18F/nsf-sbir/blob/BRANCH_NAME/README.md)

Changes proposed in this pull request:
-
-
-

/cc @relevant-people

### CODE REVIEW

File: pages/grantees/how-to-apply_phaseii.md [hunk 1]
```diff
@@ -4,29 +4,34 @@ permalink: /grantees/how-to-apply_phaseii/
...
+
+- At the start of your Phase I award, review [important information to know](http://www.nsf.gov/bfa/dias/caar/sbirrev.jsp) regarding Phase II SBIR/STTR administrative and financial capability reviews.
+- [These videos](https://www.youtube.com/playlist?list=PLGhBP1C7iCOmI1p5UtqYCXzmUL9SzSApv) also explain the review processes and how to prepare for them.
+
+## Proposal contents 
 
 - Revisit the [Merit Review Criteria]().
-- View the required contents of a [Phase II SBIR/STTR Proposal](/files/Phase%20II%20NSF%20SBIR.pdf).
-- View [guidance on determining salaries](/files/SBIR_Salary_Validation_Guide.pdf) for personnel on a Phase II grant.
+- View the required contents of a [Phase II SBIR/STTR proposal](/files/Phase%20II%20NSF%20SBIR.pdf).
```
Comment (line47) [inline pages/grantees/how-to-apply_phaseii.md]: This link 404s

File: pages/grantees/how-to-apply_phaseii.md [hunk 2]
```diff
@@ -4,29 +4,34 @@ permalink: /grantees/how-to-apply_phaseii/
...
+- View [guidance on determining salaries](/files/SBIR_Salary_Validation_Guide.pdf) for Phase II personnel.
```
Comment (line47) [inline pages/grantees/how-to-apply_phaseii.md]: This link 404s

File: pages/grantees/how-to-apply_phaseii.md [hunk 3]
```diff
@@ -4,29 +4,34 @@ permalink: /grantees/how-to-apply_phaseii/
 layout: secondary-narrow
```
Comment (line47) [inline pages/grantees/how-to-apply_phaseii.md]: We might want to change the URL to something like `/grantees/how-to-apply-phase-2/` as that is a bit more readable and doesn't mix underscores with dashes.

Comment (kategarklavs) [inline pages/grantees/how-to-apply_phaseii.md]: Ah! OK, I'll update that.

File: pages/grantees/how-to-apply_phaseii.md (hunk 2 above)
Comment (kategarklavs) [inline pages/grantees/how-to-apply_phaseii.md]: I noticed that too — I'll see if I can find the correct link. Barring that, what's our best option?

File: pages/grantees/how-to-apply_phaseii.md (hunk 1 above)
Comment (kategarklavs) [inline pages/grantees/how-to-apply_phaseii.md]: Yep, I noticed that, as well — I'll see if I can find a functional link.

File: pages/grantees/how-to-apply_phaseii.md (hunk 3 above)
Comment (kategarklavs) [inline pages/grantees/how-to-apply_phaseii.md]: Done!

File: pages/grantees/how-to-apply_phaseii.md (hunk 1 above)
Comment (kategarklavs) [inline pages/grantees/how-to-apply_phaseii.md]: OK, updated!

File: pages/grantees/how-to-apply_phaseii.md (hunk 2 above)
Comment (kategarklavs) [inline pages/grantees/how-to-apply_phaseii.md]: OK, I think I found the proper doc & I just updated the link.

File: pages/grantees/how-to-apply_phaseii.md (hunk 3 above)
Comment (line47) [inline pages/grantees/how-to-apply_phaseii.md]: The permalink is still showing `/grantees/how-to-apply_phaseii/` for me, did it get updated?

Comment (kategarklavs) [inline pages/grantees/how-to-apply_phaseii.md]: @line47 Hey! I did update it yesterday, but maybe the PR wasn't merged? I shall check and get back to you.

Comment (gemfarmer) [inline pages/grantees/how-to-apply_phaseii.md]: This has been fixed

**Pull request merged**